code/game/turfs/simulated/outdoors/grass.dm

```diff
@@ -39,7 +39,7 @@ var/list/grass_types = list(
 	grass_chance = 5
 	var/tree_chance = 2
 
-	animal_chance = 0.5
+	animal_chance = 0 //VOREStation Edit
```

Comment:
Could probably make that change on /virgo3b subtype of grass (assuming its not created via the define thing) or in _vr file otherwise.

Reply:
looks like the subtype doesn't even exist in the code.
![kuva](https://user-images.githubusercontent.com/13697337/80925681-84c1e100-8d9a-11ea-89e9-5ba2d1bd0f94.png)